.github/workflows/build.yml

```diff
@@ -0,0 +1,128 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  build-gcc-i386:
+    name: GCC (i386)
+    runs-on: ubuntu-latest
```

Comment:
Linux and windows: use explicit versions, to prevent from unexpected changes, ...?

Reply:
1. Why sticking on older OS versions? Lets have support for older builders like Visual Studio 2015, not needed to support older Operating Systems too.. Perfect will be to run these jobs on both VS 2015 and latest (2019 for now)
2. Windows have only one version to choose from: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners

Reply:
> older OS

I did not ask for 'old', I asked for 'explicit'...

> Windows have only one version

Yeah, it is confusing, as
https://help.github.com/en/actions/reference/software-installed-on-github-hosted-runners
lists S16 and S19...

Reply:
OK, 2016 lives again: https://github.com/actions/virtual-environments/issues/68#issuecomment-602652021